GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+* [@jlipps](https://github.com/jlipps)
+* [@joshbruning](https://github.com/joshbruning)
+* [@juangj](https://github.com/juangj)
+* [@julianharty](https://github.com/julianharty)
+* [@krosenvold](https://github.com/krosenvold)
+* [@lmtierney](https://github.com/lmtierney)
+* [@lukeis](https://github.com/lukeis)
+* [@mach6](https://github.com/mach6)
+* [@mtscout6](https://github.com/mtscout6)
+* [@nirvdrum](https://github.com/nirvdrum)
+* [@p0deje](https://github.com/p0deje)
+* [@santiycr](https://github.com/santiycr)
+* [@sevaseva](https://github.com/sevaseva)
+* [@shs96c](https://github.com/shs96c)
+* [@titusfortner](https://github.com/titusfortner)
+* [@tourdedavetner](https://github.com/tourdedave)
+* [@twalpole](https://github.com/twalpole)
+
+Selenium Site & Docs:
+* [@ahunsberger](https://github.com/ahunsberger)
+* [@ddavison](https://github.com/ddavison)
+* [@diemol](https://github.com/diemol)
+* [@lukeis](https://github.com/lukeis)
+* [@manoj9788](https://github.com/manoj9788)
+
+Docker-Selenium:
+* [@ddavison](https://github.com/ddavison)
+* [@diemol](https://github.com/diemol)
+* [@elgalu](https://github.com/elgalu)
+* [@kayabendroth](https://github.com/kayabendroth)
+* [@mtscout6](https://github.com/mtscout6)
+* [@WillAbides](https://github.com/WillAbides)
+
+### Technical Leadership Committee (TLC)
+
+The technical decisions and roadmap of the Selenium project are governed by
+a Technical Leadership Committee (TLC) which is responsible for a high-level
+technical guidance of the project.
+
+The TLC has final authority over this project including:
+
+* Technical direction
+* Repository hosting
+* Contribution policy (shared with the PLC)
+
+TLC seats are not time-limited. There is no fixed size of the TLC. The TLC
+should be of such a size as to ensure adequate coverage of important areas
+of expertise balanced with the ability to make decisions efficiently.
+
+The TLC may add additional members to the TLC by a standard TLC motion.
+
+A TLC member may be removed from the TLC by voluntary resignation, or by a stand
+ard TLC motion.
+
+No more than 1/3 of the TLC members may be affiliated with the same employer.
+If removal or resignation of a TLC member, or a change of employment by a TLC
+member, creates a situation where more than 1/3 of the TLC membership shares an
+employer, then the situation must be immediately remedied by the resignation or
+removal of one or more TLC members affiliated with the over-represented employer(s).
+
+TLC members have additional responsibilities over and above those of a Committer.
+These responsibilities ensure the project has a technical viability and sustainability
+in a smooth way. TLC members are expected to review code contributions, approve
+changes to this document, and manage the copyrights within the project outputs.
+
+TLC members fulfill all requirements of Committers, and also:
+
+* May merge external pull requests for accepted issues upon reviewing and approving
```

Comment:
How does this differ from the committers' rights here? Is it because they don't need to escalate to more appropriate owners? If so, it may be worth calling that out explicitly.

Also, what exactly does "external" mean here?

Reply:
An external PR is one that comes from (normally) a fork.

Regarding the difference between TLC members and committers rights: yes, the difference is that a TLC should not need to escalate to someone else when merging a PR, but having common sense (e.g. the change is easy enough that anyone can understand it and merge it). Maybe this is too detailed and perhaps we do not need this level of detail, nevertheless, I pushed a change that might be too wordy but it is also more clear.

Please let me know what you think about the change.

Reply:
The permissions thing makes sense, thanks :)

I'm still not sure on external - I'd maybe just remove the word?

Reply:
Sure, it will be more understandable. I will remove the word.